master/getting-started/kubernetes/installation/hosted/calico.yaml

```diff
@@ -260,10 +260,12 @@ spec:
         - name: cni-net-dir
           hostPath:
             path: /etc/cni/net.d
-        # Mount in the etcd TLS secrets.
+        # Mount in the etcd TLS secrets with mode 440.
+        # See https://kubernetes.io/docs/concepts/configuration/secret/
         - name: etcd-certs
           secret:
             secretName: calico-etcd-secrets
+            defaultMode: 288
```

Comment:
YAML supports octal (via prefix `0`), could you do `0440`?

Do you need group access, could this be limited to `0400`, even?